x/likenft/keeper/msg_server_mint_nft_test.go

```diff
@@ -810,3 +818,103 @@ func TestMintNFTUserNotOwner(t *testing.T) {
 	// Check mock was called as expected
 	ctrl.Finish()
 }
+func TestMintNFTNoSupply(t *testing.T) {
```

Comment:
We need another case for 0 max supply and >0 existing token count

Reply:
Added new test case for this.